README.md

```diff
@@ -291,19 +294,37 @@ Default value: `30`
 
 #### remove-stale-when-updated
 
-Automatically remove the stale label when the issues or the pull requests are updated (based on [GitHub issue](https://docs.github.com/en/rest/reference/issues) field `updated_at`) or commented.
+Automatically remove the stale label when the issues or the pull requests are updated (based on [GitHub issue](https://docs.github.com/en/rest/reference/issues) field `updated_at`).
 
 Default value: `true`
 
 #### remove-issue-stale-when-updated
 
-Override [remove-stale-when-updated](#remove-stale-when-updated) but only to automatically remove the stale label when the issues are updated (based on [GitHub issue](https://docs.github.com/en/rest/reference/issues) field `updated_at`) or commented.
+Override [remove-stale-when-updated](#remove-stale-when-updated) but only to automatically remove the stale label when the issues are updated (based on [GitHub issue](https://docs.github.com/en/rest/reference/issues) field `updated_at`).
 
 Default value: unset
 
 #### remove-pr-stale-when-updated
 
-Override [remove-stale-when-updated](#remove-stale-when-updated) but only to automatically remove the stale label when the pull requests are updated (based on [GitHub issue](https://docs.github.com/en/rest/reference/issues) field `updated_at`) or commented.
+Override [remove-stale-when-updated](#remove-stale-when-updated) but only to automatically remove the stale label when the pull requests are updated (based on [GitHub issue](https://docs.github.com/en/rest/reference/issues) field `updated_at`).
+
+Default value: unset
+
+#### remove-stale-when-commented
+
+Automatically remove the stale label when the issues or the pull requests are commented.
+
+Default value: `true`
+
+#### remove-issue-stale-when-commented
+
+Override [remove-stale-when-commented](#remove-stale-when-commented) but only to automatically remove the stale label when the issues are commented.
+
+Default value: unset
+
+#### remove-pr-stale-when-commented
+
+Override [remove-stale-when-commented](#remove-stale-when-commented) but only to automatically remove the stale label when the pull requests are commented.
 
 Default value: unset
```

Comment:
Can you also add the required permissions?

Reply:
ff07a00c816c11d5975df558f6aad6a63981cb28